src/chatd.cpp

```diff
@@ -2165,6 +2165,12 @@ void Connection::execCommand(const StaticBuffer& buf)
                 CHATDS_LOG_WARNING("%s: recv REJECT of %s: id='%s', reason: %hu",
                     ID_CSTR(chatid), Command::opcodeToStr(op), ID_CSTR(id), reason);
                 auto& chat = mChatdClient.chats(chatid);
+                if (op == OP_ADDREACTION || op == OP_DELREACTION)
+                {
+                    chat.onReactionReject(id);
+                    break;
+                }
```

Comment:
The following `if` could become an `else if` now, couldn't it?

Reply:
Adjusted in commit: fcdaee475abafb7540c9d8964946bfbb8ae7bf14